### PR TITLE
Clarify primacy of data model vs. syntax

### PR DESCRIPTION
**Pull request by jricher**

### CODE REVIEW

**Comment by awoie** [comment]
I also support the PR.

**Comment by jricher** [comment]
> I approve this PR, but see many uses of the lower-case "must"
> While I do not have a problem with the non-normative use of the word "must," others may, so it may be a good idea to re-word those sentences.

That's a fair criticism and I'd be fine with rewording "must" as "needs to" as appropriate.

**Comment by msporny** [comment]
> re: IPR failure

There is just an issue w/ mapping @jricher's Github account to his W3C account. He is clearly in the group representing SecureKey and as such there are no IPR concerns:

https://www.w3.org/2000/09/dbwg/details?group=98922&public=1&order=org

I'm ignoring the IPR warning as it's a non-issue. Ensuring /cc Chairs and W3C Staff (can we fix his affiliation?) -- @burnburn @stonematt @ashimura ... if not, I'm going to merge (when this PR is ready) anyway. Stop me if you disagree.

**Comment by msporny** [comment]
Holding on merging this PR until @jricher applies the editorial changes suggested in the conversation.

**Comment by stonematt** [comment]
VCWG Teleconference Resolution: https://www.w3.org/2019/04/09-vcwg-minutes.html#resolution04

RESOLUTION: The specification is clear wrt. using JWT and JWS with the VC Data Model. The VCWG acknowledges that a W3C Recommendation for RDF Dataset Canonicalization, Linked Data Proofs, and Linked Data Signatures does not exist, and that the Verifiable Credentials Data Model specification, as well as a number of Linked Data Working Groups at W3C, would benefit from such a set of standards. The VCWG strongly recommends the creation of a W3C Working Group to create

File: index.html [hunk 1]
```diff
@@ -511,8 +511,14 @@ <h3>Use Cases and Requirements</h3>
 <a>Issuers</a> can issue revocable <a>verifiable credentials</a>.
           </li>
           <li>
-<a>Verifiable credentials</a> and <a>verifiable presentations</a> must be
-serializable in one or more interoperable machine-readable data formats.
+<a>Verifiable credentials</a> and <a>verifiable presentations</a> has to be
```
Comment (TallTed) [inline index.html]: "has to be" -> "have to be"

File: index.html [hunk 2]
```diff
@@ -511,8 +511,14 @@ <h3>Use Cases and Requirements</h3>
...
+serializable in one or more machine-readable data formats. The serialization
+and de-serialization process has to be deterministic, bi-directional, and lossless.
```
Comment (TallTed) [inline index.html]: "process has to" -> "processes have to"

File: index.html [hunk 3]
```diff
@@ -2856,13 +2862,20 @@ <h3>Authorization</h3>
...
 <a href="#core-data-model"></a>, <a href="#basic-concepts"></a>, and
-<a href="#advanced-concepts"></a>) is realized in JSON and JSON-LD. Although
-syntactic mappings are provided for these two syntaxes only, applications and
-services can use any other data representation syntax, such as XML, YAML, or
-CBOR, that is capable of expressing the data model.
+<a href="#advanced-concepts"></a> is the canonical structural
+representation of a <a>verifiable credential</a> or <a>verifiable presentation</a>. 
+All serializations are representations of that data model in a specific 
+format. This section specifies how the data model is realized in JSON-LD and plain JSON. 
+Although syntactic mappings are provided for these two syntaxes only, applications and
```
Comment (TallTed) [inline index.html]: "for these two syntaxes only" -> "only for these two syntaxes"

File: index.html [hunk 4]
```diff
@@ -3066,8 +3092,11 @@ <h5>JWT Encoding</h5>
 JOSE header parameters, JWT registered <a>claim</a> names, or contained in the
 JWS signature part. If no explicit rule is specified, <a>properties</a> are
 encoded in the same way as with a standard <a>verifiable credential</a>, and are
-added to the <code>vc</code> <a>property</a> of the JWT. The following
-paragraphs describe these encoding rules.
+added to the <code>vc</code> <a>property</a> of the JWT. As with all JWTs,
+the JWS based signature of a <a>verifiable credential</a> represented in the JWT syntax is
```
Comment (TallTed) [inline index.html]: "JWS based" -> "JWS-based"

File: index.html [hunk 5]
```diff
@@ -3066,8 +3092,11 @@ <h5>JWT Encoding</h5>
 JOSE header parameters, JWT registered <a>claim</a> names, or contained in the
```
Comment (TallTed) [inline index.html]: I'm not sure I understand the three possibilities in the MUST. I *think* "encoded as standard JOSE header parameters, JWT registered" -> "encoded as standard JOSE header parameters, encoded as JWT registered" but if that's not what was meant, then some other word-smithing is needed.

File: index.html (hunk 2 above)
Comment (jricher) [reply index.html]: I think this is more correct in the singular as it's referring to "The process which consists of both serialization and deserialization" and not two unrelated processes.

File: index.html (hunk 3 above)
Comment (jricher) [reply index.html]: This specific language was incorporated unchanged from the previous version of the specification (see the "before" side of the diff), so I'll leave it to the editors to change that as they see fit. Either is grammatically correct.

File: index.html (hunk 5 above)
Comment (jricher) [reply index.html]: This comment refers to text not altered by this pull request. If there are additional changes to be executed here, they should be addressed in a separate commit.

File: index.html (hunk 4 above)
Comment (jricher) [reply index.html]: Got it, thanks.

File: index.html (hunk 2 above)
Comment (TallTed) [reply index.html]: Hm.  How about "The process of serialization and/or de-serialization has to be"?

File: index.html (hunk 3 above)
Comment (TallTed) [reply index.html]: Unfortunately, grammatical correctness does not always deliver clarity, which I think is increased by the relocated "only".  It does indeed go to the editors (named for attention raising -- @msporny @grantnoble @burnburn @dlongley).

File: index.html (hunk 5 above)
Comment (TallTed) [reply index.html]: OK.  https://github.com/w3c/vc-data-model/issues/552

File: index.html (hunk 3 above)
Comment (dlongley) [inline index.html]:
```suggestion
Although syntactic mappings are provided for only these two syntaxes, applications and
```

File: index.html (hunk 2 above)
Comment (jricher) [reply index.html]: A little wibbly for my tastes but it's understandable, so I'll incorporate it.

**Pull request merged**